common/app/routes/Challenges/redux/completion-epic.js

```diff
@@ -114,9 +115,14 @@ function submitBackendChallenge(type, state, { solution }) {
       })
     );
     */
+    return Observable.empty();
+  }
+  if (type === types.submitChallenge.toString()) {
     const { id } = challengeSelector(state);
     const { username } = userSelector(state);
     const csrfToken = csrfSelector(state);
+    const { solution } = getValues(state.form.BackEndChallenge);
```

Comment:
This should be a selector. Reaching into state is couples this epic with the shape of the state